### PR TITLE
make the database and connection #8

### PR DESCRIPTION
**Pull request by amanshawar**

### CODE REVIEW

**Review by Samaamro20 — APPROVED** [review]

**Comment by jema28** [comment]
@amanshawar Is the database up on heroku?

File: src/database/db_build.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+const fs = require('fs');
+const db_connection = require('./db_connection');
+const build_file = fs.readFileSync(__dirname + '/db_build.sql','utf8');
+db_connection.query(build_file,(error,result) => {
```
Comment (jema28) [inline src/database/db_build.js]: You're not using result here so it isn't necessary to pass in it in here

File: src/database/db_build.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+CREATE TABLE IF NOT EXISTS users (
+    id          SERIAL        PRIMARY KEY,
+    name        TEXT          NOT NULL,
+    email       VARCHAR(200)  NOT NULL,
+    password    VARCHAR(32)   NOT NULL
+);
+
+INSERT INTO users (name,email,password) VALUES ('samar','samar@gmail.com','samar123');
+
+-- Table books
```
Comment (jema28) [inline src/database/db_build.sql]: I would remove these comments (-- Table user etc) as it's clear what the tables are from their names.

**Review by jema28 — CHANGES_REQUESTED** [review]

File: src/database/db_build.js (hunk 1 above)
Comment (amanshawar) [reply src/database/db_build.js]: i remove result

File: src/database/db_build.sql (hunk 2 above)
Comment (amanshawar) [reply src/database/db_build.sql]: i remove the comments 
and the insert it's just for testing

and we will put the website in heroku

**Review by Hadeel-Salamin — APPROVED** [review]

**Review by jema28 — APPROVED** [review]
All tested and works well! Happy to merge 👍

**Pull request merged**